### PR TITLE
SolidityScan into solidity compiler

### PR DESCRIPTION
**Pull request by Aniket-Engg**
- Option to run classic static analysis and solidity scan now added in Solidity compiler plugin
- Result of Solidity scan appears in Remix terminal

### CODE REVIEW

**Comment by LianaHus** [comment]
- The titles you use for btns sounds more like a tooltip to me. can we get rid of "run"?
- we need icons for newly added btns. SA has one, for Sol scan you can ask them maybe

**Comment by Aniket-Engg** [comment]
> The titles you use for Btns sound more like a tooltip to me. can we get rid of "run"?

I disagree with this. That will make it less user friendly. Also other buttons are having `Publish`

**Comment by Aniket-Engg** [comment]
> The first btns promises to run analytics, but it is enabling the plugin only. It should enable with await and run on the current compiled contract

No APIs are exposed in SSA plugin to run analysis like that. For now, I move users to SSA plugin to run analysis there

**Comment by Aniket-Engg** [comment]
> we need icons for newly added bins. SA has one, for Sol scan, you can ask them maybe

Add icon for SSA, asked for solscan one

File: apps/remix-ide/src/app/tabs/locales/en/solidity.json [hunk 1]
```diff
@@ -52,6 +52,19 @@
 
   "solidity._comment_contract-selection.tsx": "libs/remix-ui/solidity-compiler/src/lib/contract-selection.tsx",
   "solidity.publishOn": "Publish on",
+  "solidity.runStaticAnalysis": "Run Static Analysis",
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/locales/en/solidity.json:55]: I 'd propose `Run Remix Analysis`

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/tabs/locales/en/solidity.json]: updated

**Comment by yann300** [comment]
LGTM!

**Comment by LianaHus** [comment]
- you have issues with dark icons in dark theme
- we may consider moving icons to the left as it is everywhere else in the ide
- only the last one has no icon. we could add

1. `<i class="fa-regular fa-memo-circle-info"></i>`
or 
3. `<i class="fa-regular fa-memo-pad"></i>`

**Review by LianaHus — APPROVED** [review]

**Pull request merged**